pyramid/router.py

```diff
@@ -213,7 +214,7 @@ def invoke_subrequest(self, request, use_tweens=False):
             try:
                 extensions = self.request_extensions
                 if extensions is not None:
-                    request._set_extensions(extensions)
+                    apply_request_extensions(request, extensions=extensions)
```

Comment:
`self.request_extensions` is just the interface that `apply_request_extensions` queries anyways, do we need to pass them in?